spec/models/pawn_spec.rb

```diff
@@ -61,28 +61,28 @@
     end
 
     it 'should return false if moving one space forward when destination square is occupied' do
-      pawn = FactoryGirl.create(:pawn, x_position: 4, y_position: 4, is_white: true)
-      FactoryGirl.create(:rook, x_position: 4, y_position: 5, is_white: false)
+      pawn = FactoryGirl.create(:pawn, x_position: 4, y_position: 4, is_white: false)
+      FactoryGirl.create(:rook, x_position: 4, y_position: 5, is_white: true)
       expect(pawn.valid_move?(4, 5)).to eq false
```

Comment:
If the pawn being moved is black, then a forward move for the pawn at (4, 4) would be (4, 3), not (4, 5). Since black pieces now start at top of board after you switched starting positions for black and white. So this test needs to use those coords before it can test if pawn move forward is considered invalid if destination is occupied.